### ISSUE TITLE
About SURVIVOR merge parameter

### ISSUE THREAD

**Issue by fantasticair** [fritzsedlazeck/SURVIVOR#136]
Hello,
     I have a doubt about the parameters `./SURVIVOR merge sample_files 1000 2 1 1 0 30 sample_merged.vcf`, I don't know what does the 0 before 30 stands for? What is the difference between the results of 1 and 0 here?

Best wishes ~

**Comment by carolynzy** [comment]
I think this parameter is disabled, as said in the instruction. So it doesn't matter what value you use just keep a position for it will be fine.

**Comment by fritzsedlazeck** [comment]
yes that parameter is currently disabled. I didnt want to change the interface for people that have implemented a script around it or so.. 
Thanks
Fritz
